File: C/Iniciante/1115quadrante.c
#include <stdio.h>
int main() {
    int x,y;
    scanf("%d%d", &x, &y);
    while (1){
        if(x==0 || y==0)break;
        if (x > 0 && y > 0) {
            //q1
            printf("primeiro\n");
        } else if (x < 0 && y > 0) {
            //q2
            printf("segundo\n");
        } else if (x < 0 && y < 0) {
            //q3
            printf("terceiro\n");
        } else if (x > 0 && y < 0) {
            //q4
            printf("quarto\n");
        }
        scanf("%d%d", &x, &y);
    }
    return 0;
}

File: Python/iniciante/1115quadrante.py
x, y = input().split(" ")
x = int(x)
y = int(y)
while 1:
    if x == 0 and y == 0:
        break
    if (x > 0) and (y > 0):
        print("primeiro")
    elif (x < 0) and (y > 0):
        print("segundo")
    elif (x < 0) and (y < 0):
        print("terceiro")
    elif (x > 0) and (y < 0):
        print("quarto")
    x, y = input().split(" ")
    x = int(x)
    y = int(y)
